data/presets.yaml

```diff
@@ -16,6 +16,16 @@ parsers:
 - sqlite/chrome_66_cookies
 - sqlite/skype
 ---
+name: ios
+description: Preset for iOS.
+parsers:
+- ios_lockdownd
+- ios_logd
+- ios_mobile_installation_log
```

Comment:
Other iOS plugins:
- ios_application_privacy
- ios_netusage (in progress)
- ios_screentime (already merged)
- ios_sysdiag (merged)
- ips_parser/apple_crash_log (in progress)

-